packages/mobile/src/components/premium-track-purchase-drawer/PurchaseSuccess.tsx

```diff
@@ -72,6 +85,19 @@ export const PurchaseSuccess = ({ track }: { track: UserTrackMetadata }) => {
         handle={handle}
         size='large'
       />
+      <Button
+        onPress={onClose}
+        title={messages.viewTrack}
+        variant='commonAlt'
+        styles={{
+          root: styles.viewTrack,
+          text: styles.viewTrackText,
+          // @ts-ignore icon has ViewStyle prop for some reason, which does not understand the 'color' property
+          icon: styles.viewTrackIcon
```

Comment:
wondering what prop this icon style should be instead of `ViewStyle`

Reply:
You could use `IconProps` to pass it a color directly?